modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -0,0 +1,873 @@
+
+---
+
+=== emergency_disable_data_transforms
+
+Override the cluster property xref:reference/properties/cluster-properties.adoc#data_transforms_enabled[`data_transforms_enabled`] and disable Wasm-powered data transforms. This is an emergency shutoff button.
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== empty_seed_starts_cluster
+
+Controls how a new cluster is formed. All brokers in a cluster must have the same value.
+
+<<seed_servers,See how the `empty_seed_starts_cluster` setting works with the `seed_servers` setting>> to form a cluster.
+
+TIP: For backward compatibility, `true` is the default. Redpanda recommends using `false` in production environments to prevent accidental cluster formation.
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== kafka_api
+
+IP address and port of the Kafka API endpoint that handles requests.
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Default:* `127.0.0.1:9092`
+
+---
+
+=== kafka_api_tls
+
+Transport Layer Security (TLS) configuration for the Kafka API endpoint.
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Default:* `null`
+
+---
+
+=== memory_allocation_warning_threshold
+
+Log messages that contain a larger memory allocation than specified.
+
+*Unit:* bytes
+
+*Optional:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `131073` (128_kib + 1)
+
+---
+
+=== node_id
+
+A number that uniquely identifies the broker within the cluster. If `null` (the default value), Redpanda automatically assigns an ID. If set, it must be non-negative value.
+
+CAUTION: The `node_id` property must not be changed after a broker joins the cluster.
+
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Type:* integer
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Default:* `null`
+
+---
+
+=== rack
+
+A label that identifies a failure zone. Apply the same label to all brokers in the same failure zone. When xref:./cluster-properties.adoc#enable_rack_awareness[enable_rack_awareness] is set to `true` at the cluster level, the system uses the rack labels to spread partition replicas across different failure zones.
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Default:* `null`
+
+---
+
+=== recovery_mode_enabled
+
+If `true`, start Redpanda in xref:manage:recovery-mode.adoc[recovery mode], where user partitions are not loaded and only administrative operations are allowed.
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== rpc_server
+
+IP address and port for the Remote Procedure Call (RPC) server.
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Default:* `127.0.0.1:33145`
+
+---
+
+=== rpc_server_tls
+
+TLS configuration for the RPC server.
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+---
+
+=== seed_servers
+
+List of the seed servers used to join current cluster. If the seed_server list is empty the node will be a cluster root and it will form a new cluster.
```

Comment:
```suggestion
List of the seed servers used to join current cluster. If the `seed_servers` list is empty the node will be a cluster root and it will form a new cluster.
```